helm_deploy/probation-offender-search/templates/deployment.yaml

```diff
@@ -46,15 +46,15 @@ spec:
               protocol: TCP
           livenessProbe:
             httpGet:
-              path: /ping
+              path: /health/ping
```

Comment:
should really be using `/liveness` and `/readiness` here instead

Reply:
yeas, agreed - next pr